src/test/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -685,4 +693,205 @@ describe("Graph - Axes", () => {
+            ).toBe("Some X Label");
+            expect(
+                fetchElementByClass(styles.axisLabelY2).querySelector("text")
+                    .textContent
+            ).toBe("Some Y2 Label");
+            done();
+        });
+    });
+    describe("Axes labels with different axes lengths", () => {
+        beforeEach(() => {
+            graph.destroy();
+            axisDefault.y2 = {
+                show: true,
+                label: "Some Y2 Label",
+                lowerLimit: 0,
+                upperLimit: 20
+            };
+        });
+        afterEach(() => {
+            axisDefault.y2 = "";
+        });
+        it("Truncates if too long", (done) => {
+            graphContainer.setAttribute("style", "width: 800px; height: 200px");
+            const labelAxisObj = utils.deepClone(axisDefault);
+            labelAxisObj.y.label = largeLabel;
+            labelAxisObj.y2.label = largeLabel;
+            labelAxisObj.x.label = largeLabel;
+            dimension.height = 800;
+            graph = new Graph(
+                Object.assign({ dimension }, getAxes(labelAxisObj), dimension)
+            );
+            expect(
+                fetchElementByClass(styles.axisLabelX).querySelector("text")
+                    .textContent
+            ).toBe(
+                "Project long display value which is only for testing, Project long display value which ..."
+            );
+            expect(
+                fetchElementByClass(styles.axisLabelY).querySelector("text")
+                    .textContent
+            ).toBe(
+                "Project long display value which is only for testing, Project long display value which is only for t..."
+            );
+            expect(
+                fetchElementByClass(styles.axisLabelY2).querySelector("text")
+                    .textContent
+            ).toBe(
+                "Project long display value which is only for testing, Project long display value which is only for t..."
+            );
+            done();
+        });
+        it("Does not truncate if not long", (done) => {
+            graph = new Graph(getAxes(axisDefault));
+            expect(
+                fetchElementByClass(styles.axisLabelY).querySelector("text")
+                    .textContent
+            ).toBe("Some Y Label");
+            expect(
+                fetchElementByClass(styles.axisLabelX).querySelector("text")
+                    .textContent
+            ).toBe("Some X Label");
+            expect(
+                fetchElementByClass(styles.axisLabelY2).querySelector("text")
+                    .textContent
+            ).toBe("Some Y2 Label");
+            done();
+        });
+    });
+    describe("Adds clickHandler for Label", () => {
+        const onClickPrimaryFunctionSpy = sinon.spy();
+        const onClickSecondaryFunctionSpy = sinon.spy();
+        const onClickThirdFunctionSpy = sinon.spy();
+        const onClickFourthFunctionSpy = sinon.spy();
```

Comment:
Can we move these constants to respective `it` block?

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/242/files#diff-11aa0aaf6db553889f349765e88f7462R1000)